templates/zabbix_server.conf.erb

```diff
@@ -516,3 +527,4 @@ LoadModulePath=<%= @loadmodulepath %>
 <% if @tlskeyfile %>TLSKeyFile=<%= @tlskeyfile %><% end %>
 
 <% end %>
+
```

Comment:
Remove extra newline. Would be good to also remove it from `zabbix_proxy.conf.erb`.
```suggestion
```

Reply:
Should be done in both files in latest commit.